test/end-to-end/thirdpartyemailpassword.test.js

```diff
@@ -141,6 +144,40 @@ describe("SuperTokens Third Party Email Password", function () {
             assert.deepStrictEqual(pathname, "/dashboard");
         });
 
+        it("should show error message on sign up with duplicate email", async function () {
+            await toggleSignInSignUp(page);
+
+            const requestHandler = (request) => {
+                if (request.url().includes(EMAIL_EXISTS_API) && request.method() === "GET") {
+                    return request.respond({
+                        status: 200,
+                        headers: {
+                            "access-control-allow-origin": TEST_CLIENT_BASE_URL,
+                            "access-control-allow-credentials": "true",
+                        },
+                        body: JSON.stringify({
+                            status: "OK",
+                            exists: true,
+                        }),
+                    });
+                }
+
+                return request.continue();
+            };
+
+            try {
+                await page.setRequestInterception(true);
+                page.on("request", requestHandler);
+                await setInputValues(page, [{ name: "email", value: "john.doe@supertokens.io" }]);
+            } finally {
+                page.off("request", requestHandler);
+                await page.setRequestInterception(false);
+            }
+
```

Comment:
also need to click on the sign up button and make sure that it causes an error in the email form field about email already existing.